### ISSUE TITLE
Felzárkoztató tantárgyak 

### ISSUE THREAD

**Issue by botalaszlo** [battila7/changes#3]
Én anno jártam a Debreceni Egyetemre PTI-re, most máshol csinálok egy MI MSc-t.
A legnagyobb hátránya, hogy nincs felzárkoztató tantárgy Matematikából. (Óbudai Egyetemen a BSc-eseknek van például a nyáron a felvételi után lehetőségük erre, illetve ott van egy másik egyetem, mint az ELTE, ahol anno tartottak felmérőt is igénybe lehetett venni, vagy az Eszterházy Egyetem, ahol van Matematika Praktikum tantárgy). Egy-egy ilyen tantárgy nagyon jó lenne, hogy közös szintre hozná a hallgatókat.

Debreceni Egyetem matematika oktatása szerintem ettől függetlenül nagyon jó, erősnek mondanám, de ha az alapoknál elvérzik a diák, akkor csak szenvedni fog.
Ugyanezt lehetne programozásból is, lenne 1-1 szájbarágos tantárgy az elején, szerintem nagyon jó lenne, utána már tényleg a diák dolga legyen, hogy mennyire készül fel.
Ezeket azért tudnám javasolni, mert öcsém például az OE-n van Mechatronikán is ez bevált nekik, hogy nem egyből középsuli után a mélyvízbe dobták. Nem volt penge matekból, most sem az, de nem ilyed meg az egyetemi matek anyagtól, sőt egész jól veszi az akadályt.

**Comment by battila7** [comment]
Sajnos már nem emlékszem, hogy a Kalkulus 1 meg a Diszkrét matematika 1 hol veszi fel a fonalat, de az biztos, hogy a heterogén középiskolai tudást érdemes lenne valamiféle közös szintre felhozni az első félévben.

Emellett pedig egy nagyon-nagyon erős Intro to CS kellene, mert a friss hallgatók között ugyanúgy van olyan, aki még sosem programozott és foglalkozott számítógéppel érdemi szinten, mint aki már évek óta írogat programokat hobbiból és már tud ezt-azt.

**Comment by botalaszlo** [comment]
Pontosan, és itt azokra kell gondolni, akik például gimnáziumból jöttek.
